File: src/main/java/de/hsrm/mi/abschlussarbeit/requestProcessingSystem/support/enums/ExpertiseLevel.java
package de.hsrm.mi.abschlussarbeit.requestProcessingSystem.support.enums;

import lombok.Getter;

@Getter
public enum ExpertiseLevel {
    BEGINNER(1),
    INTERMEDIATE(2),
    ADVANCED(3),
    EXPERT(4);

    private final int points;

    ExpertiseLevel(int points) {
        this.points = points;
    }

}

File: frontend/Workstation/src/documentTypes/types/ExpertiseLevel.ts
export enum ExpertiseLevel {
  BEGINNER = "BEGINNER",
  INTERMEDIATE = "INTERMEDIATE",
  ADVANCED = "ADVANCED",
  EXPERT = "EXPERT",
}

export const ExpertiseLevelLabel: Record<ExpertiseLevel, string> = {
  [ExpertiseLevel.BEGINNER]: 'Anfänger',
  [ExpertiseLevel.INTERMEDIATE]: 'Mittel',
  [ExpertiseLevel.ADVANCED]: 'Fortgeschritten',
  [ExpertiseLevel.EXPERT]: 'Experte',
}
